library/mageia

```diff
@@ -1,24 +1,24 @@
 Maintainers: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
 GitRepo: https://github.com/juanluisbaptiste/docker-brew-mageia.git
 
-Tags: 8,latest
-GitCommit: 067e6df19c568e101a42dc71b77f6a2de5992c70
+Tags: 9,latest
+GitCommit: 2365d6c0c7c87fb7e120348bac4c3f1bd1e21af7
 GitFetch: refs/heads/dist
 Architectures: amd64, arm32v7, arm64v8
 amd64-Directory: dist/8/x86_64
 arm32v7-Directory: dist/8/armv7hl
 arm64v8-Directory: dist/8/aarch64
```

Comment:
This isn't quite correct. This is still pointing at the `8` directories on the `9` image. I think it just needs to swap to `dist/9/...` on all the `*-Directory` values.

```suggestion
amd64-Directory: dist/9/x86_64
arm32v7-Directory: dist/9/armv7hl
arm64v8-Directory: dist/9/aarch64
```

Reply:
Ooops! fixed now :smile: